Add missing includes in various places.

tutorials/common/common_vaapi.cpp:100:17: error: implicit instantiation of undefined template 'std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >'
    std::string adapterPath = DRI_PATH;
                ^
/usr/include/c++/v1/iosfwd:209:32: note: template is declared here
    class _LIBCPP_TEMPLATE_VIS basic_string;
                               ^
tutorials/common/common_vaapi.cpp:107:57: error: no member named 'to_string' in namespace 'std'
        std::string curAdapterPath = adapterPath + std::to_string(nodeIndex + i);
                                                   ~~~~~^
tutorials/common/common_vaapi.cpp:107:21: error: implicit instantiation of undefined template 'std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >'
        std::string curAdapterPath = adapterPath + std::to_string(nodeIndex + i);
                    ^
/usr/include/c++/v1/iosfwd:209:32: note: template is declared here
    class _LIBCPP_TEMPLATE_VIS basic_string;
                               ^

--- tutorials/common/common_vaapi.cpp.orig	2019-10-30 19:56:10 UTC
+++ tutorials/common/common_vaapi.cpp
@@ -22,6 +22,7 @@
 #include <drm.h>
 #include <drm_fourcc.h>
 #include <map>
+#include <string>
 
 #include "common_vaapi.h"
 
